references/cli/inferless-volume.mdx

```diff
@@ -1,23 +1,20 @@
 ---
-title: "inferless volumes"
+title: 'inferless volumes'
 ---
 
-
-Manage Inferless volumes 
+Manage Inferless volumes
 
 ```console
 $ inferless volume COMMAND [OPTIONS] [ARGS]...
 ```
 
 **Commands**:
 
-* `create`: Create a new volume.
-* `cp`: Add a file or directory to a volume.
-* `ls`: List files and directories within a volume.
-* `rm`: Specify the Inferless path to the file.
-* `list`: List all existing volumes.
-* `select`: Select a volume for the current Inferless configuration file.
-
+- `create`: Create a new volume.
+- `cp`: Add a file or directory to a volume.
+- `ls`: List files and directories within a volume.
+- `rm`: Specify the Inferless path to the file.
+- `list`: List all existing volumes.
```

Comment:
Is delete alo there in cli ?

Reply:
deleteting a volume is not present. deleting file or dir from the volume is present.